### PR TITLE
replacing msdn link

### PR DESCRIPTION
**Pull request by mairaw**
Related to #2037 

[Internal review link](https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.build.tasks.windows.generatetemporarytargetassembly?branch=pr-en-us-4012)

### CODE REVIEW

File: xml/Microsoft.Build.Tasks.Windows/GenerateTemporaryTargetAssembly.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>Implements the <c>GenerateTemporaryTargetAssembly</c> task. Use the <c>GenerateTemporaryTargetAssembly</c> element in your project file to create and execute this task. For usage and parameter information, see [GenerateTemporaryTargetAssembly](http://msdn.microsoft.com/library/92b6539c-6897-45e0-8989-0c234bbfe782).</summary>
+    <summary>Implements the <c>GenerateTemporaryTargetAssembly</c> task. Use the <c>GenerateTemporaryTargetAssembly</c> element in your project file to create and execute this task. For usage and parameter information, see [GenerateTemporaryTargetAssembly](/visualstudio/msbuild/generatetemporarytargetassembly-task).</summary>
```
Comment (jzeferino) [inline xml/Microsoft.Build.Tasks.Windows/GenerateTemporaryTargetAssembly.xml]: How can I validate if the link is working?

**Comment by BillWagner** [comment]
@mairaw I get a 404 on the updated link.  Did I miss something?

**Comment by mairaw** [comment]
The cross-repo links don’t work on staging. You need to change to master branch on the target url. 

@jzeferino you can add docs.microsoft.com in front of that path and see if the link is right.

**Comment by jzeferino** [comment]
I can confirm the link is working properly. @mairaw

**Review by BillWagner — APPROVED** [review]
LGTM.  I'll :shipit: now.

**Pull request merged**